src/jumpstart/saga.ts

```diff
@@ -0,0 +1,199 @@
+      eventName: ['ERC20Claimed'],
+      logs,
+    })
+
+    for (const {
+      address,
+      args: { token: tokenAddress, amount },
+    } of parsedLogs) {
+      const tokenId = getTokenId(networkId, tokenAddress)
+
+      const token = tokensById[tokenId]
+      if (!token) {
+        Logger.error(TAG, 'Claimed unknown tokenId', tokenId)
+        continue
+      }
+
+      const value = new BigNumber(amount.toString()).shiftedBy(-token.decimals).toFixed()
+
+      yield* put(
+        addStandbyTransaction({
+          __typename: 'TokenTransferV3',
+          type: TokenTransactionTypeV2.Received,
+          context: {
+            id: transactionHash,
+          },
+          transactionHash,
+          networkId,
+          amount: {
+            value,
+            tokenAddress,
+            tokenId,
+          },
+          address,
+          metadata: {},
+        })
+      )
+
+      ValoraAnalytics.track(JumpstartEvents.jumpstart_claimed_token, {
+        networkId,
+        tokenAddress,
+        value: Number(value),
+      })
+    }
+  }
+}
+
+export function* dispatchPendingERC721Transactions(
+  networkId: NetworkId,
+  transactionReceipts: TransactionReceipt[]
+) {
+  for (const { transactionHash, logs } of transactionReceipts) {
+    const parsedLogs = parseEventLogs({
+      abi: walletJumpstart.abi,
+      eventName: ['ERC721Claimed'],
+      logs,
+    })
+
+    for (const {
+      args: { token: contractAddress, tokenId },
+    } of parsedLogs) {
+      try {
+        const network = networkIdToNetwork[networkId]
+        const tokenUri = (yield* call([publicClient[network], 'readContract'], {
+          address: contractAddress,
+          abi: parseAbi(['function tokenURI(uint256 tokenId) returns (string)']),
+          functionName: 'tokenURI',
+          args: [tokenId],
+        })) as string
+
+        const response = yield* call(fetchWithTimeout, tokenUri)
+        const metadata = (yield* call([response, response.json])) as NftMetadata
+
+        yield* put(
+          addStandbyTransaction({
+            __typename: 'NftTransferV3',
+            type: TokenTransactionTypeV2.NftReceived,
+            context: {
+              id: transactionHash,
+            },
+            transactionHash,
+            networkId,
+            nfts: [
+              {
+                tokenId: tokenId.toString(),
+                contractAddress,
+                tokenUri,
+                metadata,
+                media: [
+                  {
+                    raw: metadata?.image,
+                    gateway: metadata?.image,
+                  },
+                ],
+              },
+            ],
+          })
+        )
+
+        ValoraAnalytics.track(JumpstartEvents.jumpstart_claimed_nft, {
+          networkId,
+          contractAddress,
+          tokenId: tokenId.toString(),
+        })
+      } catch (error) {
+        Logger.error(TAG, 'Error adding pending NFT transaction', error)
```

Comment:
same here, i wonder if we should downgrade these to .warn since the pending transactions is a nice to have visual feedback rather than core functionality?